content/applications/inventory_and_mrp/inventory/routes/strategies/storage_category.rst

```diff
@@ -0,0 +1,175 @@
+==================
+Storage categories
+==================
+
+A *storage category* is used with :doc:`putaway rules <putaway>`, as an extra location attribute to
+automatically propose optimal storage locations for products.
+
+Follow these steps to complete the setup:
+
+#. Enable the feature
```

Comment:
Suggestion to add a reference to this as well, to really be clear on which feature to enable :)
```suggestion
#. :ref:`Enable the Storage Category feature <inventory/routes/enable-storage-categories>`
```